CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- check-marathon-task.rb: solved a bug which prevented to alert when number of instances were retrieved from Marathon
```

Comment:
This should go under `### Fixed` but I will do that for you this time since it is an obvious fix.